internals/processes.py

```diff
@@ -478,18 +482,67 @@ def process_to_dict(process):
   ]
 
 
+FEATURE_ROLLOUT_STAGE = ProcessStage(
+      'Start feature rollout',
+      'Lock in shipping milestone. '
+      'Create feature flag for the feature. '
+      'Create policies to enable/disable and control the feature. '
+      'Finalize docs and announcements and start rolling out the feature.',
+      [PI_ROLLOUT_MILESTONE,
+       PI_ROLLOUT_PLATFORMS,
+       PI_ROLLOUT_DETAILS,
+       PI_ENTERPRISE_POLICIES,
+      ],
+      [],
+      [],
+      core_enums.INTENT_SHIP, core_enums.INTENT_SHIP)
+
+ENTERPRISE_STAGES = [
```

Comment:
Please add a comment here saying that these are the stages for a feature that has the  enterprise feature_type.